src/openni2/DeviceDriver.cpp

```diff
@@ -500,7 +500,43 @@ namespace Freenect2Driver
             WriteMessage("Opening device " + std::string(uri));
```

Comment:
Do you need `#include <libfreenect2/packet_pipeline.h>`? (Just asking, I don't know.)